packages/ui/src/lib/components/post.svelte

```diff
@@ -1,79 +1,90 @@
 <script lang="ts">
+	import Card from '$lib/components/grid-card.svelte'
 	import { formatDateAndTime } from '$lib/utils/format'
-
 	import type { Post } from '$lib/stores/post'
 
 	let cls: string | undefined = undefined
-	export { cls as class }
 	export let post: Post
 </script>
 
-<div class={`root ${cls}`}>
+<Card on:click>
```

Comment:
Does this actually work? I don't see where this `click` property is coming from.